core/src/main/java/io/lionweb/lioncore/java/serialization/ProtoBufSerialization.java

```diff
@@ -0,0 +1,345 @@
+package io.lionweb.lioncore.java.serialization;
+
+import io.lionweb.lioncore.java.language.LionCoreBuiltins;
+import io.lionweb.lioncore.java.model.ClassifierInstance;
+import io.lionweb.lioncore.java.model.impl.ProxyNode;
+import io.lionweb.lioncore.java.self.LionCore;
+import io.lionweb.lioncore.java.serialization.data.*;
+import io.lionweb.lioncore.java.serialization.data.MetaPointer;
+import io.lionweb.lioncore.protobuf.*;
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class ProtoBufSerialization extends AbstractSerialization {
+
+  /** This has specific support for LionCore or LionCoreBuiltins. */
+  public static ProtoBufSerialization getStandardSerialization() {
```

Comment:
Maybe externalize this standard setup? It's the same for every serialization as far as I see.

Reply:
Done: I have now introduced the class SerializationProvider and moved all those methods there, sharing the code where needed

Reply:
(Sorry, I did not push it to the right branch, now I did)